core/controllers/suggestion.py

```diff
@@ -523,7 +579,7 @@ def _upload_suggestion_images(request, suggestion, filenames):
     # TODO(#10513): Find a way to save the images before the suggestion is
     # created.
     for filename in filenames:
-        image = request.get(filename)
+        image = files.get(filename) if files else None
```

Comment:
As per the docstring we are expecting files to be a dict, right? in that case why do we need the `if files` check?

Reply:
Done. I have removed this extra check and some unnecessary validation error part in this function because we are already  performing validation of images in the schema.